README.md

```diff
@@ -194,6 +194,14 @@ Make sure to list bootstrap in externals in `webpack.config.base.js` or the app
 externals: ['bootstrap']
 ```
 
+### Debugging
+
+You can debug the main process by following one of these approaches:
+- [VSCode](https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_attaching-to-nodejs)
+- [Chrome](https://nodejs.org/en/docs/guides/debugging-getting-started/#inspector-clients)
```

Comment:
Would be great to add a link for IntelliJ too (or maybe better just to https://nodejs.org/en/docs/guides/debugging-getting-started/) without favoring a single editor :)

Reply:
@DominikGuzei could you please confirm if that guide works for Electron? I haven't added it because i don't use IntelliJ editor hence Chrome linked.